src/components/record/Record.jsx

```diff
@@ -198,6 +263,28 @@ class Record extends React.Component {
     );
   }
 
+  _renderModal() {
+    const { showModal, invalidQuestions, incompleteQuestions } = this.state;
+    const questionsToShow = invalidQuestions.length > 0 ? invalidQuestions : incompleteQuestions;
+
+    let modalMessage = "";
+    if (questionsToShow === invalidQuestions) {
+      modalMessage = "Some questions are invalid. Correct them to save:";
```

Comment:
@blcham When we agree on the actual message, I will add i18n

Reply:
"Some answers are invalid. Correct them to save:"